src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -67,14 +67,15 @@ class SaveInProgressIntro extends React.Component {
     } else if (renderSignInMessage) {
       alert = renderSignInMessage(prefillEnabled);
     } else if (prefillEnabled && !verifyRequiredPrefill) {
+      const { retentionPeriod } = this.props;
       alert = (
         <div>
           <div className="usa-alert usa-alert-info schemaform-sip-alert">
             <div className="usa-alert-body">
               <strong>If you’re signed in to your account, your application process can go more smoothly. Here’s why:</strong><br/>
               <ul>
                 <li>We can prefill part of your application based on your account details.</li>
-                <li>You can save your form in progress, and come back later to finish filling it out. You have 60 days from the date you start or update your application to submit the form. After 60 days, the form won’t be saved, and you’ll need to start over.</li>
+                <li>You can save your form in progress, and come back later to finish filling it out. You have {retentionPeriod} from the date you start or update your application to submit the form. After {retentionPeriod}, the form won’t be saved, and you’ll need to start over.</li>
```

Comment:
I decided to only make the actual period a prop (instead of the whole string) because I feel that there's some value to standardizing the messaging as much as possible.